TicTacToe/Logic/Game.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+
+namespace TicTacToe.Logic
+{
+    public sealed class Game // Sealed because we want to keep a private constructor
+    {
+        public const int BoardDimensions = 3;
+        private int attemptsCount = 0;
```

Comment:
Naming: why is this member name not _attemptsCount given the other member names do start with underscore?

Consistency :)